mozilla-lint-rules/src/test/java/org/mozilla/fenix/lintrules/ContextCompatDetectorTest.kt

```diff
@@ -0,0 +1,159 @@
+package org.mozilla.fenix.lintrules
```

Comment:
nit - license (haha maybe the linter would have caught this :P )

Reply:
Oh lol, I missed it (I guess I don't see the top of my files often haha)

Reply:
Hm, I'm not getting the `LicenseDetector` warning on test files, maybe there's something I need to configure?